testing/main_integration_test.go

```diff
@@ -128,7 +128,7 @@ type Package struct {
 }
 
 func getPackages(t *testing.T) ([]string, error) {
-	resp, err := http.Get("http://localhost:8080/search")
+	resp, err := http.Get("http://localhost:8080/search?experimental=true&kibana.version=8.0.0")
```

Comment:
I'm probably misunderstanding some but I thought we were going to be switching the packages and the endpoint package to kibana version `^7.9.0` which would be excluded here right?

So if I need to release a new endpoint package for 7.9 in production that has the kibana version `^7.9.0` would CI here pick it up?

Reply:
Did anyone consider a switch on the Kibana side? User could select between stable and experimental packages.

Reply:
@mtojek Yes, that was the initial plan for 7.9. I was now delayed as we enabled experimental packages by default as most of the packages are in experimental.

@jonathan-buttner You are correct with the above and I will open an issue around what version of Kibana we should test against. Currently we test against 8.0.0-SNAPSHOT so I would say the above is correct. Getting this PR in first to have production on a more recent version of the registry and then follow up.